src/main/resources/twitter-demo/static/js/tweetbook.js

```diff
@@ -755,7 +755,15 @@ function drawHashtag(tag_count) {
  **/
 function drawTweets(message) {
   $.each(message, function (i, d) {
-    $('#tweets tr:last').after('<tr><td>' + d.uname + '<br/>' + d.tweet + '</td></tr>');
+    // Using JSONP
+    var url = "https://api.twitter.com/1/statuses/oembed.json?id=" + d.id;
+    $.ajax({
```

Comment:
Shall we store the request in AsterixDB? Since I saw someone stores all the html in the database. Or we just store it as a file?